mix.exs

```diff
@@ -10,7 +10,16 @@ defmodule Nimble.MixProject do
       compilers: [:phoenix, :gettext] ++ Mix.compilers(),
       start_permanent: Mix.env() == :prod,
       aliases: aliases(),
-      deps: deps()
+      deps: deps(),
+      test_coverage: [tool: ExCoveralls],
+      preferred_cli_env: [
+        lint: :test,
+        coverage: :test,
+        coveralls: :test,
+        "coveralls.detail": :test,
+        "coveralls.post": :test,
```

Comment:
Since we don't use `coveralls.detail` and `coveralls.post` for now in the aliases, could these two be removed?

Reply:
Thanks. Fixed: 74139f9